website/docs/d/oracle_db_system.html.markdown

```diff
@@ -0,0 +1,121 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_db_system"
+description: |-
+  Gets information about an existing DB System.
+---
+
+# Data Source: azurerm_oracle_db_system
+
+Use this data source to access information about an existing DB System.
```

Comment:
what `DB` stands for? If it stands for `database`, could you use `database` to replace all `DB`s because abbreviations are not allowed